tutorials/04 - DistortionEffectDemo.md

```diff
@@ -0,0 +1,138 @@
+# 04 - Distortion Effect
+
+In this tutorial, we're going to analyze the structure of an audio effect application, by applying distortion to a looping audio file.
```

Comment:
Also probably state what key topics covered are here such as:
- Adding customisation to an Edit through use of a custom Plugin class
- Writing this plugin class
- Registering it with the Engine so it can be added to an Edit's PluginList
- Adding it to a track
- Writing a custom ValueSource to bind a Slider/Value to an AutomatableParameter